cli/lib/exec/spawn.js

```diff
@@ -155,7 +155,7 @@ module.exports = {
 
         const { onStderrData } = overrides
         const envOverrides = util.getEnvOverrides(options)
-        const electronArgs = []
+        const electronArgs = ['--ozone-platform=auto']
```

Comment:
I believe that the tests here will need to be updated for this change: https://github.com/cypress-io/cypress/blob/develop/cli/test/lib/exec/spawn_spec.js#L117

Reply:
Is that part of ci/circleci: unit-tests workflow? If it is, how actually added more xml test reports to the repo?